libc/src/__support/threads/linux/CndVar.cpp

```diff
@@ -27,7 +29,7 @@ int CndVar::wait(Mutex *m) {
 
   CndWaiter waiter;
   {
-    MutexLock ml(&qmtx);
+    cpp::lock_guard ml(qmtx);
```

Comment:
I think so, hh. `cpp::lock_guard` is more generic. Guess we can remove `MutexLock` later on, since they serve the same functionality and `MutexLock` is type specific.

Reply:
oh, I forgot to request that `MutexLock` get removed in c4a3d184db5fdffe798208b8281dfe944616f9ed.

Reply:
Mind splitting the removal of that and the fixup to libc/src/__support/threads/linux/CndVar.cpp of my mistake to a distinct PR from this one?

Reply:
ok